client/src/app/app.component.html

```diff
@@ -6,6 +6,7 @@
     <button mat-menu-item (click)="onPositionListClick()">Position list</button>
     <button mat-menu-item (click)="onAddPositionClick()">Add position</button>
   </mat-menu>
+  <button mat-raised-button (click)="onRedactorListClick()">Redactors</button>
```

Comment:
Zmień proszę typ przycisku na `mat-button` i dodaj przejście do listy jako podmenu. Tak jak w przypadku pozycji. Za chwilę będziemy implementować dodawanie redaktora, więc będzie to przydatne :)